src/kirby/components/list/list.component.ts

```diff
@@ -39,6 +39,7 @@ export class ListSectionHeaderDirective {}
 export class ListComponent implements OnInit {
   @Input() items: any[];
   @Input() getSectionName?: (item: any) => string;
+  @Input() lines = false;
```

Comment:
I can see you found inspiration in Ionic's [list](https://ionicframework.com/docs/api/list#properties) 👍

Reply:
Yep :-)